skaffold.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: skaffold/v1
+kind: Config
+build:
+  artifacts:
+    - image: lab-partner
+      context: ./
+  local: {}
+
+deploy:
+  helm:
+    releases:
+      - name: lab-partner
+        chartPath: charts/lab-partner
+        wait: true
+        setValueTemplates:
+          image.name: lab-partner
```

Comment:
does this work?  `setValueTemplates` is supposed to parse environment variables and pass the `--set` flags to helm with the values.  how does this work with the dynamically built artifact on line 5?